.github/workflows/push.yaml

```diff
@@ -281,6 +298,13 @@ jobs:
             "${root_dir}"/OBS-Studio-${{ steps.check.outputs.version }}-Ubuntu-x86_64-dbsym.ddeb
           mv -v "${ubuntu_sources_name}/"obs-studio-*-sources.tar.gz \
             "${root_dir}"/OBS-Studio-${{ steps.check.outputs.version }}-Sources.tar.gz
+          mv -v "${windows_installer_name}/"OBS-Studio-*-x64.exe \
+            "${root_dir}"/OBS-Studio-${{ steps.check.outputs.version }}-Windows-x64-Installer.exe
```

Comment:
If we're trying to match the existing pattern the Windows Installer name is:
* OBS-Studio-30.1.2-Full-Installer-x64.exe

If we're intentionally changing the name pattern here, that's fine with me. I'm not sure if this name is used by other jobs.

Reply:
This shouldn't be used by other jobs, Steam uses the ZIPs, so does the patch generation. I think it's time to retired the `Full` part from the installer given that we haven't shipped without a browser source in what, 6 years?